contracts/drafts/ERC20Snapshot.sol

```diff
@@ -10,6 +10,11 @@ import "../token/ERC20/ERC20.sol";
  * inspired by Jordi Baylina's MiniMeToken to record historical balances
  * Snapshots store a value at the time a snapshot is taken (and a new snapshot id created), and the corresponding
  * snapshot id. Each account has individual snapshots taken on demand, as does the token's total supply.
+ *
+ * To make a snapshot, call the `snapshot` function, which will emit the `Snapshot` event and return a snapshot id.
+ * To get the total supply from a snapshot, call the function `totalSupplyAt` with the snapshot id.
```

Comment:
This phrasing may be misleading, a snapshot doesn't have supply or balances, but rather supply and account balance can be queried for a particular snapshot (returning the values at the time the snapshot was taken).

Reply:
I think you are getting into implementation details with that explanation. From a black box point of view, I see there is snapshot with id, supply and balances.
But this is just a suggestion, feel free to modify to your liking.

Reply:
I think @nventuro means that people might misunderstand that a snapshot is like a token with its own transfers. I don't think this will be a problem, the word Snapshot communicates a static thing IMO.